taskcluster/docker/toolchain-build/Dockerfile

```diff
@@ -19,7 +19,11 @@ RUN apt-get update -qq \
                           tar \
                           libxml2 \
                           libhunspell-dev \
-                          bc \
+                          bc  \
+                          autoconf \
```

Comment:
@bhearsum these are the four packages required by hunspell that we need to add on the generic worker

Reply:
ok, we don't need it with the toolchain, but I'd still leave if for the future when we fully switch to Docker. Then we can remove the toolchain.